em/emSizeField.cc

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright (C) 2011 Scientific Computation Research Center
+ *
+ * This work is open source software, licensed under the terms of the
+ * BSD license as described in the LICENSE file in the top-level directory.
+ */
+#include <iostream>
+#include <cstdlib>
+
+#include "apfElement.h"
+#include "crv.h"
+#include "crvShape.h"
+#include <apfCavityOp.h>
+
+#include "em.h"
+using namespace std;
```

Comment:
not used in this file. should be removed.

Reply:
Removed.